Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/menu/ui/TestMenuFragmentViewModel.kt

```diff
@@ -53,6 +54,7 @@ class TestMenuFragmentViewModel @AssistedInject constructor(
             DscTestFragment.MENU_ITEM,
             BoosterTestFragment.MENU_ITEM,
             DccTicketingTestFragment.MENU_ITEM,
+            CCLTestFragment.MENU_ITEM,
```

Comment:
Maybe put this one a bit higher instead so that we don't always need to scroll down that much ;)

Reply:
done